client/src/components/App.jsx

```diff
@@ -1,6 +1,33 @@
 import React, { Component } from 'react';
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      reviews: [],
+      productcode: 1
+    };
+    this.changeReviews = this.changeReviews.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchReviews();
+  }
+
+  // HTTP Request Handlers
+  fetchReviews() {
+    let url = `/api/models/${this.state.productcode}/reviews`;
+    fetch(url)
+      .then(response => response.json())
+      .then(reviewsData => this.changeReviews(reviewsData))
+      .catch(err => console.error(err));
+  }
+
+  // Set State
+  changeReviews(reviews) {
+    this.setState({ reviews: reviews });
```

Comment:
In the sprit of ES6, I wonder if you can just say  `this.setState({reviews});`. I'm actually not sure if you can do that, let me know